### DISCUSSION TITLE
Rotary as a button

### DISCUSSION THREAD

**Discussion by diablonrw** [shanteacontrols/OpenDeck#257] [General]
Hi, can you take the Rotary entrances as buttons? So with each race between 0-65(127) in the cw direction and in ccw towards 0-1?

**Comment by paradajz** [comment]
I'm really not sure what are you asking.

**Reply by diablonrw** [reply]
My English is very bad. 

I use a translator. 

I have one rotary encoder and also have the Rotary Enabled. That works too. But when I turn the Rotary, the Open Deck L sends "cw" 1 or "ccw" 65(127) on the note. My question, is it possible for each step of Rotary first the value (ccw: 65 or cw:1) and then after x milliseconds value (0) until the next step?

Almost the function of two buttons.

big thx

**Reply by paradajz** [reply]
No, that is not possible. You can use the encoder in continuous CC mode, however.

**Comment by paradajz** [comment]
Encoders now support configuration of two notes, one for each direction.
